src/js/experimental/media-rendition-listbox.js

```diff
@@ -0,0 +1,155 @@
+  <style>
+    media-chrome-option {
+      white-space: var(--media-rendition-listbox-white-space, nowrap);
+    }
+  </style>
+`;
+
+/**
+ * @attr {string} mediarenditionselected - (read-only) Set to the enabled rendition.
+ * @attr {string} mediarenditionlist - (read-only) Set to the rendition list.
+ *
+ * @cssproperty --media-rendition-listbox-white-space - `white-space` of playback rate list item.
+ */
+class MediaRenditionListbox extends MediaChromeListbox {
+  #autoOption;
+  #renditionList = [];
+  #prevState;
+
+  static get observedAttributes() {
+    return [
+      ...super.observedAttributes,
+      MediaUIAttributes.MEDIA_RENDITION_LIST,
+      MediaUIAttributes.MEDIA_RENDITION_SELECTED,
+    ];
+  }
+
+  constructor() {
+    super({ slotTemplate });
+
+    const autoOption = document.createElement('media-chrome-option');
+
+    autoOption.part.add('option');
+    autoOption.value = 'auto';
+    autoOption.textContent = 'Auto';
+    this.#autoOption = autoOption;
+  }
+
+  attributeChangedCallback(attrName, oldValue, newValue) {
+
+    if (attrName === MediaUIAttributes.MEDIA_RENDITION_SELECTED && oldValue !== newValue) {
+      this.value = parseRenditionList(newValue)[0];
+
+    } else if (attrName === MediaUIAttributes.MEDIA_RENDITION_LIST && oldValue !== newValue) {
+
+      this.#renditionList = parseRenditionList(newValue);
+      this.#render();
+    }
+
+    super.attributeChangedCallback(attrName, oldValue, newValue);
+  }
+
+  connectedCallback() {
+    this.addEventListener('change', this.#onChange);
+
+    super.connectedCallback();
+  }
+
+  disconnectedCallback() {
+    this.removeEventListener('change', this.#onChange);
+
+    super.disconnectedCallback();
+  }
+
+  get mediaRenditionList() {
+    return this.#renditionList;
+  }
+
+  set mediaRenditionList(list) {
+    this.removeAttribute(MediaUIAttributes.MEDIA_RENDITION_LIST);
+
+    this.#renditionList = list;
+    this.#render();
+  }
+
+  get mediaRenditionSelected() {
+    return this.mediaRenditionList.find(({ id }) => id == this.value);
+  }
+
+  set mediaRenditionSelected(rendition) {
+    this.removeAttribute(MediaUIAttributes.MEDIA_RENDITION_SELECTED);
+
+    this.value = rendition?.id;
+  }
+
+  #render() {
+    if (this.#prevState === JSON.stringify(this.mediaRenditionList)) return;
+    this.#prevState = JSON.stringify(this.mediaRenditionList);
+
+    const renditionList = this.mediaRenditionList
+      .sort((a, b) => b.height - a.height);
+
+    const container = this.shadowRoot.querySelector('slot');
+    container.textContent = '';
+
+    if (!container.contains(this.#autoOption)) {
+      container.append(this.#autoOption);
+    }
+
+    let isAuto = !this.mediaRenditionSelected;
+    if (isAuto) {
+      this.#autoOption.setAttribute('aria-selected', 'true');
+      this.#autoOption.setAttribute('tabindex', '0');
+    } else {
+      this.#autoOption.setAttribute('aria-selected', 'false');
+      this.#autoOption.setAttribute('tabindex', '-1');
```

Comment:
Why is auto getting a -1 tabindex here? I'm assuming we still want auto tab selectable? Probably worth a comment.

Reply:
I removed this from this PR. I'll address it in the other one.
I think it is needed once you select an option, it's how it's done.
Set the selected option to tabindex=0 and the rest of the options to tabindex=-1